cmd/gitsign-credential-cache/main.go

```diff
@@ -47,31 +49,53 @@ func main() {
 		os.Exit(0)
 	}
 
-	user, err := os.UserCacheDir()
-	if err != nil {
-		log.Fatalf("error getting user cache directory: %v", err)
-	}
+	var connChan = make(chan net.Conn)
+	if *systemdFlag {
+		listeners, err := activation.Listeners()
+		if err != nil {
+			log.Fatalf("error getting systemd listeners: %v", err)
+		}
+		for _, l := range listeners {
```

Comment:
https://pkg.go.dev/github.com/coreos/go-systemd/activation#Listeners says that some of these listeners may return nil - we probably want to filter those out.

Reply:
Fixed